lib/idx/remediators/Base/VerifyAuthenticator.ts

```diff
@@ -38,6 +45,16 @@ export class VerifyAuthenticator extends Remediator {
     this.authenticator = getAuthenticator(remediation);
   }
 
+  getNextStep(context?: IdxContext): NextStep {
+    const nextStep = super.getNextStep(context);
+    const authenticatorEnrollments = context?.authenticatorEnrollments?.value;
```

Comment:
For webauthn verification we need to get enrolled authenticators (with type webauthn) and extract credential ids.
This data is stored in context.
Realization of context is taken from https://github.com/okta/okta-auth-js/pull/1026